src/specfem2d.cpp

```diff
@@ -163,9 +163,31 @@ void execute(const std::string &parameter_file, const std::string &default_file,
   specfem::compute::assembly assembly(
       mesh, quadrature, sources, receivers, setup.get_seismogram_types(),
       setup.get_t0(), dt, nsteps, max_seismogram_time_step,
-      setup.get_simulation_type());
+      setup.get_simulation_type(), true);
```

Comment:
The assign property boolean should come from setup class. If we are planning to set it to true always then we dont need a boolean

Reply:
Thanks, I forgot to change this. Now updated.